app/Helpers/UserRepository.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: bendozy
+ * Date: 8/19/15
+ * Time: 12:40 PM
+ */
+
+namespace ChopBox\helpers;
+
+use ChopBox\User;
+
+class UserRepository {
+
+		public function findUserByEmail($email){
+
+				return $user = User::where('email', '=', $email)->first();
+		}
```

Comment:
Why do you have the UserRepository file in Helpers folder?. Not cool. You can create a Repository folder and put it there. Please Fix!